dive_core/data_core.cpp

```diff
@@ -570,6 +532,60 @@ void CaptureMetadataCreator::FillEventStateInfo(EventStateInfo::Iterator event_s
     FillHardwareSpecificStates(event_state_it);
 }
 
+//--------------------------------------------------------------------------------------------------
+void CaptureMetadataCreator::FillResolveOrClearEventStateInfo(
+EventStateInfo::Iterator event_state_it)
+{
+    uint32_t rb_resolve_gmem_buffer_base_reg_offset = GetRegOffsetByName(
+    "RB_RESOLVE_GMEM_BUFFER_BASE");
+    if (m_state_tracker.IsRegSet(rb_resolve_gmem_buffer_base_reg_offset))
+    {
+        RB_RESOLVE_GMEM_BUFFER_BASE rb_resolve_gmem_buffer_base_reg;
+        rb_resolve_gmem_buffer_base_reg.u32All = m_state_tracker.GetRegValue(
+        rb_resolve_gmem_buffer_base_reg_offset);
+        event_state_it->SetResolveBaseGmem(rb_resolve_gmem_buffer_base_reg.u32All);
```

Comment:
just curious, how come the base address is a 32bits value? i thought all address are 64bits

Reply:
oh, i just read your later change, it is an offset not a real address. Wondering if would be better to call it SetResolveBaseGmemOffset